src/core/Akka.Tests/Delivery/ConsumerControllerRetryConfirmSpecs.cs

```diff
@@ -0,0 +1,75 @@
+﻿// -----------------------------------------------------------------------
+//  <copyright file="ConsumerControllerRetryConfirmSpecs.cs" company="Akka.NET Project">
+//      Copyright (C) 2009-2024 Lightbend Inc. <http://www.lightbend.com>
+//      Copyright (C) 2013-2024 .NET Foundation <https://github.com/akkadotnet/akka.net>
+//  </copyright>
+// -----------------------------------------------------------------------
+
+using System.Threading.Tasks;
+using Akka.Actor;
+using Akka.Configuration;
+using Akka.Delivery;
+using Akka.Util;
+using FluentAssertions;
+using FluentAssertions.Extensions;
+using Xunit;
+using Xunit.Abstractions;
+using static Akka.Tests.Delivery.TestConsumer;
+
+namespace Akka.Tests.Delivery;
+
+public class ConsumerControllerRetryConfirmSpecs : TestKit.Xunit2.TestKit
+{
+    public static readonly Config Config = @"
+        akka.reliable-delivery.consumer-controller {
+        flow-control-window = 20
+        resend-interval-min = 1s
```

Comment:
Might want to lower this given that you're waiting for up to 1.5 seconds - just to provide a bigger window in the event of racy tests

Reply:
ok, will do